specification/marketplacecatalog/resource-manager/Keys/stable/2023-01-01-preview/discoveryKeys.json

```diff
@@ -0,0 +1,279 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Discovery API Keys Management",
+    "description": "This API is designed to streamline the process of generating designated keys for accessing all Marketplace Catalog APIs. By utilizing a simple POST operation, users can effortlessly generate a unique API key, granting  access to Marketplace Catalog APIs from version 2023-01-01 and beyond.",
+    "version": "2023-01-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Marketplace/keys/{apiKeyAlias}/create": {
+      "post": {
+        "tags": [
+          "CreateApiKey"
+        ],
+        "summary": "Generates an API key, granting access to the full range of Marketplace Catalog APIs. Please note that a valid subscription is required to utilize this functionality.",
+        "operationId": "CreateApiKey_Post",
+        "x-ms-examples": {
+          "Create_Key": {
+            "$ref": "./examples/CreateKey.json"
+          }
+        },
+        "parameters": [
+          {
+            "type": "string",
+            "name": "apiKeyAlias",
+            "in": "path",
+            "required": true,
+            "description": "Alias for new API Key. This alias must be unique and contain only alphanumeric characters separated by dashes."
+          },
+          {
+            "type": "string",
+            "name": "x-ms-client-tenant-id",
+            "x-originalName": "tenantId",
+            "description": "The tenant associated with the subscription.",
+            "in": "header",
+            "required": true,
+            "x-nullable": true
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
```

Comment:
Use the `subscriptionId` from the common definition, 
`../../../../../common-types/resource-management/v5/types.json#/parameters/....`

Reply:
Fixed